components/popover/popover.vue

```diff
@@ -607,6 +628,18 @@ export default {
    *     METHODS    *
    ******************/
   methods: {
+
+    hasIntersectedViewport (entries, observer) {
+      const dialog = entries[0];
+
+      // if dialog is cut off from the viewport
+      if (!dialog.isIntersecting) {
```

Comment:
Could be simplified to this `this.isOutsideViewport = !dialog.isIntersecting;`

Reply:
great thanks!